jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/bsm/BsmReceiver2.java

```diff
@@ -0,0 +1,106 @@
+package us.dot.its.jpo.ode.bsm;
+
+import java.net.DatagramPacket;
+import java.util.Arrays;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.oss.asn1.AbstractData;
+
+import us.dot.its.jpo.ode.OdeProperties;
+import us.dot.its.jpo.ode.j2735.dsrc.BasicSafetyMessage;
+import us.dot.its.jpo.ode.plugin.j2735.J2735Bsm;
+import us.dot.its.jpo.ode.plugin.j2735.oss.OssBsm;
+import us.dot.its.jpo.ode.plugin.j2735.oss.OssBsmPart2Content.OssBsmPart2Exception;
+import us.dot.its.jpo.ode.udp.AbstractUdpReceiverPublisher;
+import us.dot.its.jpo.ode.util.JsonUtils;
+import us.dot.its.jpo.ode.util.SerializationUtils;
+import us.dot.its.jpo.ode.wrapper.MessageProducer;
+
+public class BsmReceiver2 extends AbstractUdpReceiverPublisher {
```

Comment:
Kept the old one for comparison and to make sure I did not leave anything out. Once BsmReceiver2 tested, we should copy the code over to BsmReceiver and delete BsmReceiver2.